_posts/2018-04-11-galaxy-africa.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Galaxy Africa"
+tags: [training,talk]
+location: Cape Town, South Africa
+site: freiburg
+---
+
+<blockquote class="twitter-tweet" data-lang="fr"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/usegalaxy?src=hash&amp;ref_src=twsrc%5Etfw">#usegalaxy</a> Africa is starting!! <a href="https://twitter.com/hashtag/GalaxyAfrica?src=hash&amp;ref_src=twsrc%5Etfw">#GalaxyAfrica</a> <a href="https://t.co/6NOy10ITDk">pic.twitter.com/6NOy10ITDk</a></p>&mdash; Bérénice Batut (@bebatut) <a href="https://twitter.com/bebatut/status/981073917187100672?ref_src=twsrc%5Etfw">3 avril 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The first [Galaxy conference](http://galaxyafrica.sanbi.ac.za/) was held last week in Africa, organized by [SANBI](http://www.sanbi.ac.za/) at [University of Wester Cape](https://www.uwc.ac.za/Pages/default.aspx) in Cape Town. Björn and Bérénice participated:
+
+- On the 3rd, Björn gave the keynote talk ("Welcome to Galaxy") in the morning. During the afternoon, they gave a Galaxy introduction training
+- On the 4th, Bérénice started with the keynote talk (["Welcome to Galaxy"](http://bebatut.fr/talks/18/04_04_galaxy_africa/#/1))
+
+    <div class="multiple-img">
+        <embed src="http://bebatut.fr/talks/18/04_04_galaxy_africa/#/1" width="65%" height="300px">
```

Comment:
just noting this for myself, we'll have to either:

- wait for github to enable ssl-for-custom-domains for your domain
- or proxy this site/page behind SSL
- maybe we could use rawgit?